data/core/editor/help.cfg

```diff
@@ -280,6 +280,51 @@ To get started, refer to the <ref>dst='editor_tool_paint' text='Paint Tool'</ref
 [/topic]
 # wmllint: markcheck on
 
+# wmllint: markcheck off
+[topic]
+    id=..editor_mode_scenario
+    title= _ "Scenario Editor"
+    text= _ "The scenario editor mode adds support for some map-related WML features, such as areas and scenery items. Most scenarios will still require additional WML to be written using a different tool, the scenario editor does not support scripting the scenario’s events." + "
```

Comment:
> Most scenarios will still require additional WML to be written using a different tool, the scenario editor does not support scripting the scenario’s events.

This should be separated by a semi-colon rather than a comma.